src/xdocs/google_style.xml

```diff
@@ -1282,7 +1282,7 @@
                             <td><a name="6.2"/><a href="#6.2"><img src="images/anchor.png" alt=""/></a></td>
                             <td>
                                 <a
-                                    href="http://checkstyle.sourceforge.net/reports/google-java-style.html#s6.2-caught-exceptions">6.2 Caught exceptions: not ignored</a>
+                                    href="http://checkstyle.sourceforge.net/reports/google-java-style-20160712.html#s6.2-caught-exceptions">6.2 Caught exceptions: not ignored</a>
```

Comment:
Old version: empty catch blocks are no allowed, unless the variable name is `expected`. New version is that the variable name should _start_ with `expected`. I did a few tests and it looks like checkstyle already follows the new rule (please confirm that I'm not wrong).